### PR TITLE
Adds documentation

### PR DESCRIPTION
**Pull request by manovotny**
* Explains what blue-green deployments are
* Links to CI/CD automation
* Adds links to Vercel docs in various places
* Adds deploy button
* Adds link to template

### CODE REVIEW

**Review by cramforce — APPROVED** [review]

File: .github/workflows/cron-blue-green-deploy.yml [hunk 1]
```diff
@@ -57,22 +59,28 @@ jobs:
         run: pnpm install --no-frozen-lockfile
       - name: Install Vercel CLI
         run: npm install --global vercel
+        # Pull down the production environment variables as if were building on Vercel.
+        # https://vercel.com/docs/cli/pull
       - name: Get Vercel Environment Variables
         run: vercel pull --yes --environment=production --token=${{ secrets.VERCEL_TOKEN }}
+        # Create a "blue" version of the site, deloy it, adnd capture the deployment url.
```
Comment (AhmedBaset) [inline .github/workflows/cron-blue-green-deploy.yml:66]:
```suggestion
        # Create a "blue" version of the site, deploy it, and capture the deployment url.
```

File: .github/workflows/cron-blue-green-deploy.yml [hunk 2]
```diff
@@ -57,22 +59,28 @@ jobs:
...
       - name: Update CSS to Blue
         run: 'echo "body {background: blue;}" > ./app/blue-green.css'
       - name: Build Blue
         run: vercel build --prod --token=${{ secrets.VERCEL_TOKEN }}
       - name: Deploy Blue
         id: deploy-blue
         run: echo "url=$(vercel deploy --prebuilt --prod --token=${{ secrets.VERCEL_TOKEN }})" >> $GITHUB_OUTPUT
+        # Create a "green" version of the site, deloy it, adnd capture the deployment url.
```
Comment (AhmedBaset) [inline .github/workflows/cron-blue-green-deploy.yml:74]:
```suggestion
        # Create a "green" version of the site, deploy it, and capture the deployment url.
```

File: .github/workflows/cron-blue-green-deploy.yml (hunk 1 above)
Comment (manovotny) [reply .github/workflows/cron-blue-green-deploy.yml:66]: Addressed in https://github.com/vercel-labs/blue-green/commit/2f96c2cb3123bd9fb29565886a024eddbfb095be.

File: .github/workflows/cron-blue-green-deploy.yml (hunk 2 above)
Comment (manovotny) [reply .github/workflows/cron-blue-green-deploy.yml:74]: Addressed in https://github.com/vercel-labs/blue-green/commit/2f96c2cb3123bd9fb29565886a024eddbfb095be.

**Pull request merged**